test/test.js

```diff
@@ -962,4 +962,18 @@ asyncTest('Loading a System.registerdynamic module (not bundled)', function() {
   }).then(null, err);
 });
 
+asyncTest('Importing a script with wrong integrity fails', function() {
```

Comment:
Note this test also runs in Node, so add a selection rule like `if (typeof window != 'undefined')`.